### ISSUE TITLE
Very nice analysis!

### ISSUE THREAD

**Issue by arnoudbuzing** [lynxgav/COVID19-mitigation#1]
Someone pointed me to this very nice analysis and your paper (which was linked from CNN). Have you considered doing a writeup of the Wolfram Language (Mathematica) aspects of this analysis on https://community.wolfram.com/ ? (I think other Mathematica users would enjoy it!)

**Comment by lynxgav** [comment]
Hi Arnoud,

it is a good idea. I think the code is correct but certain parts could be
improved. Writing about it as is would be easier than working on upgrading
it.

I am on holidays. I could do it in a month or so. Is it still acceptable?

Ganna

**Comment by arnoudbuzing** [comment]
Absolutely! Totally up to you. Glad you're even considering my suggestion. Looking forward to your post when you have the time!
